### DISCUSSION TITLE
Glpi agent monitor new ticket

### DISCUSSION THREAD

**Discussion by Luesp81** [glpi-project/glpi-agentmonitor#12] [General]
Good morning,
I have a problem, when I click on new ticket it opens the GLPI page but it gives me an error: Invalid use of session ID and I have to log in manually.
how can I solve it?

I would also like to be able to open the forms page (/marketplace/formcreator/front/formlist.php) and not /front/ticket.form.php which is disabled.

Thank you

**Comment by redddcyclone** [comment]
Hi @Luesp81 

The new ticket button only opens the browser on the /front/ticket.form.php URL. If the user is not logged in, the login page will be displayed with the error you mentioned.

This is normal and expected, as the user probably should be logged in to open new tickets. If the user then logs in, they should be redirected to the new ticket page, though in your case it probably won't as you use Form Creator.

**Comment by Luesp81** [comment]
Can't I change the redirection?, I saw that the string is in the GLPI-AgentMonitor.cpp file, but I can't find the file in the installation.
Thank you

**Reply by redddcyclone** [reply]
You won't find this file in the installation. The source code is compiled into the .exe file. The only way would be to change it in code and recompile the executable by yourself.

As I said in another issue in my repo, I will be adding a settings dialog soon where you'll be able to change it.

**Reply by Luesp81** [reply]
Thanks for your reply, I would also ask you how to recompile...

Would there also be a way to log in automatically?

Thank you

**Reply by redddcyclone** [reply]
For recompiling the project, you should install Visual Studio with Visual C++, open the project file and build the project. For more info, do some search.

For logging in automatically, you should look into enabling SSO (single sign-on) and LDAP login on your GLPI instance if your computers are domain joined. If everything is set up correctly, the browser uses your Windows credentials to log in to GLPI automatically. Helping with that is out of my scope.

**Comment by vincent1890** [comment]
J'ai exactement la même demande 
J'utilise FormCreator comme énormément de monde et votre outils est génial mais étant donnée que j'utilise FormCreator comment rediriger les utilisateur sur la page des formulaire accessible a l'utilisateur
Je me demande si une redirection apache pourrais faire le travail (temporairement) !
